.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: releaser
+
+on:
+  create:
+      tags:
+  # push:
+  #   branches:
+  #     - 'release'
+  #   tags:
+  #     - 'v*'
+
+jobs:
+  build:
+    name: Build release on ${{ matrix.os }} with ${{ matrix.compiler }}.
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            compiler: gcc
+            target: Linux
+
+          - os: macos-latest
+            compiler: clang
+            target: Macos
+
+          - os: windows-latest
+            compiler: cl
+            target: Windows
+    # if: github.event_name == 'create' && github.event.ref_type == 'tag' && startsWith(github.event.ref, 'v')
+    # https://stackoverflow.com/questions/59349905/which-properties-does-github-event-in-a-github-workflow-have
+    # https://github.community/t/how-to-get-just-the-tag-name/16241/15
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      -
+        name: Cache Go modules
+        uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+            
+      # - name: Setup tmate session
+      #   uses: mxschmitt/action-tmate@v3
+
+      - name: Get the version
+        id: get_version
+        run: |
```

Comment:
get version by `go run cmd/zarb Version`

Reply:
that is not feasible at this step it's not been set so it default to `v1.0.0-beta` 
the version is set in next step via `goreleaser` action so actually we don't need this any more